src/plugins/data_views/common/fields/utils.ts

```diff
@@ -26,14 +26,22 @@ export const isMultiField = isDataViewFieldSubtypeMulti;
 export const getFieldSubtypeMulti = getDataViewFieldSubtypeMulti;
 export const getFieldSubtypeNested = getDataViewFieldSubtypeNested;
 
-const DOT_PREFIX_RE = /(.).+?\./g;
-
 /**
  * Convert a dot.notated.string into a short
  * version (d.n.string)
  */
 export function shortenDottedString(input: string): string {
-  return typeof input !== 'string' ? input : input.replace(DOT_PREFIX_RE, '$1.');
+  if (typeof input === 'string') {
+    const split = input.split('.');
```

Comment:
Can some parts be empty? For example, can input be something like `foo..bar`? 

```
const split = input.split('.').filter(Boolean);
```

Reply:
yes, field names can be anything

Reply:
https://github.com/elastic/kibana/pull/193874

Reply:
Just FYI I actually tried ingesting a document like `foo..bar.foo` and I got an error:

```
POST tmp-00001/_doc
{
  "asdf..basdf": "bar"
}

// Results in 
{
  "error": {
    "root_cause": [
      {
        "type": "document_parsing_exception",
        "reason": "[1:4] failed to parse: field name cannot contain only whitespace: ['asdf..basdf']"
      }
    ],
    "type": "document_parsing_exception",
    "reason": "[1:4] failed to parse: field name cannot contain only whitespace: ['asdf..basdf']",
    "caused_by": {
      "type": "illegal_argument_exception",
      "reason": "field name cannot contain only whitespace: ['asdf..basdf']"
    }
  },
  "status": 400
}

Reply:
I'll skip the follow up until someone can produce a field name with two dots in a row